packages/common/src/search/utils.ts

```diff
@@ -349,3 +349,25 @@ export function getResultSiteRelativeLink(
   }
   return siteRelativeLink;
 }
+
+/**
+ * Adds default predicates for item target entity
+ *
+ * @param query IQuery to search items
+ * @returns an IQuery with default predicates
+ */
+export function getItemQueryWithDefaultPredicates(query: IQuery): IQuery {
+  if (query.targetEntity === "item") {
```

Comment:
Normally it's good to do a check like this, but because the name of the function specifies that it works on item queries, you can remove this. Within hub.js directly, this function will never get called with a `query.targetEntity` other than item.

Reply:
I agree and that is a great point! I have removed the check.